docs/python/linting.md

```diff
@@ -44,10 +41,53 @@ A custom path is generally unnecessary as the Python extension resolve the path
+
+Custom arguments can be specified in the appropriate arguments setting for each linter, with each argument given as a separate item in the array.
+
+A custom path is generally unnecessary as the Python extension resolve the path to the linter based on the Python interpreter being used (see [Environments](/docs/python/environments.md)). To use a different version of a linter, specify its path in the appropriate custom path setting.
+
 The sections that follow provide additional details for those individual linters linked in the table.
 
 ## Pylint
 
+Pylint messages fall into the categories in the following table with the indicated mapping to VS Code categories. You can change the setting to change the mapping.
+
+| Pylint category | Description | VS Code category mapping | Applicable setting<br/>(python.linting.) |
+| --- | --- | --- | --- |
+| Convention (C) | Programming standard violation | Information (green underline) | pylintCategorySeverity.convention |
+| Refactor (R) | Bad code smell | Hint (light bulbs) | pylintCategorySeverity.refactor |
+| Warning (W) | pylintCategorySeverity.warning | Warning | Python-specific problems |
+| Error (E) | pylintCategorySeverity.error | Error (red underline) | Likely code bugs |
+| Fatal (F) | pylintCategorySeverity.fatal | Error | An error prevented further Pylint processing |
+
+### Default Pylint rules
+
+Python in Visual Studio code is configured by default to use a set of linting rules that are friendly to the largest number of  Python developers:
+
+- Enable all Error (E) and Fatal (F) messages.
+- Disable all Convention (C) and Refactor (R) messages.
+- Disable all Warning (W) messages except the following:
+  - unreachable (W0101): Unreachable code
+  - duplicate-key (W0109): Duplicate key %r in dictionary
+  - unnecessary-semicolon (W0301): Unnecessary semicolon
+  - global-variable-not-assigned (W0602): Using global for %r but no assignment is done
+  - unused-variable (W0612): Unused variable %r
+  - binary-op-exception (W0711): Exception to catch is the result of a binary "%s" operation
+  - bad-format-string (W1302): Invalid format string
+  - anomalous-backslash-in-string (W1401): Anomalous backslash in string
+  - bad-open-mode (W1501): "%s" is not a valid mode for open
+
+These rules are applied through the following default arguments passed to Pylint:
+
+```
+--enable=F,E,unreachable,duplicate-key,unnecessary-semicolon,global-variable-not-assigned,unused-variable,binary-op-exception,bad-format-string,anomalous-backslash-in-string,bad-open-mode
+```
+
+These arguments are passed whenever the `python.linting.pylintUseMinimalCheckers` is set to `true` (the default). If you specify a value in `pylintArgs` or use a Pylint configuration file (see the next section), then set `pylintUseMinimalCheckers` to `false`. You can set it back to `true` to easily revert to the defaults as needed.
```

Comment:
"then set `pylintUseMinimalCheckers` to `false`" -> "then `pylintUseMinimalCheckers` is implicitly set to `false`"

"You can set it back to `true` to easily revert to the defaults as needed" -> "You can set it to `true` to override these default semantics"

Reply:
Setting to true uses the defaults, not override them, yes? Your suggested edit here is contradictory, unless you say set to false to override. Is there a case where you'd purposely set to false?

Reply:
That is, can we drop that last sentence entirely?